CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.19.0]
```

Comment:
Shouldn't this be 0.18.2? They seem backward-compatible bug fixes which according to semver should be an increase of patch-level version

Reply:
right, I will update the PR